WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderWebView.java

```diff
@@ -95,10 +95,15 @@ private void init() {
             this.setWebChromeClient(mReaderChromeClient);
             this.setWebViewClient(new ReaderWebViewClient(this));
             this.getSettings().setUserAgentString(WordPress.getUserAgent());
-            this.getSettings().setDefaultFontSize(
-                    (int) (this.getSettings().getDefaultFontSize() * getResources().getConfiguration().fontScale));
-            this.getSettings().setDefaultFixedFontSize(
-                    (int) (this.getSettings().getDefaultFixedFontSize() * getResources().getConfiguration().fontScale));
+
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
```

Comment:
Hey Nick, I've added a comment (good point!) and limited the font sizing for only when the scale is not too close to 1. (Notice that it's not a good practice to do `equals` checks with floats and that's why I did it with rounding to some significant digits).

Ready for another pass at this point. Thanks!